app/src/main/java/com/kickstarter/ui/views/compose/checkout/PledgeItemizedDetails.kt

```diff
@@ -101,7 +128,8 @@ fun ItemizedRewardListContainer(
     totalBonusSupport: String,
     deliveryDateString: String = "",
     rewardsHaveShippables: Boolean = false,
-    disclaimerText: String = ""
+    disclaimerText: String = "",
+    plotSelected: Boolean
```

Comment:
We can pass in a default value of `false` here for `plotSelected`, and then it will only show when we pass in a `true` value. that way we aren't required to pass in a value for this field everywhere we call this function unless it's PLOT related

Reply:
Change added 😄